com.unity.render-pipelines.universal/ShaderLibrary/UnityInput.hlsl

```diff
@@ -124,11 +127,15 @@ CBUFFER_END
 #if defined(USING_STEREO_MATRICES)
 CBUFFER_START(UnityStereoViewBuffer)
 float4x4 unity_StereoMatrixP[2];
-float4x4 unity_StereoMatrixIP[2];
+float4x4 unity_StereoMatrixInvP[2];
 float4x4 unity_StereoMatrixV[2];
 float4x4 unity_StereoMatrixInvV[2];
 float4x4 unity_StereoMatrixVP[2];
-float4x4 unity_StereoMatrixIVP[2];
+float4x4 unity_StereoMatrixInvVP[2];
```

Comment:
same possible breakage here.

Reply:
I added `#define unity_MatrixIVP unity_MatrixInvVP ` in `deprecated.hlsl` file.

Reply:
Do you also want `unity_StereoMatrixIVP` to be #define too?